app/views/deposit/collection/_invoices_table.html.erb

```diff
@@ -44,7 +44,8 @@ $(document).ready(function() {
         }
       ],
     createdRow: function(row, data, dataIndex) {
-      $('td', row).eq(4).html('<input class="payment_amount_invoice" type="number" name="payment_amount_' + data[0] + '" id="payment_amount_' + dataIndex + '" step="any" class="form-control">')
+      var disabled = data[3] == 0 ? ' disabled' : '';
```

Comment:
`data[3]` is the amount with the currency. Remove all non-numeric characters before comparing it to 0.